Resources/Prototypes/Entities/Structures/Specific/anomalies.yml

```diff
@@ -179,3 +179,38 @@
     anomalyContactDamage:
       types:
         Radiation: 10
+
+
+- type: entity
+  id: AnomalyIce
+  parent: BaseAnomaly
+  suffix: Ice
+  components:
+  - type: Sprite
+    sprite: Structures/Specific/Anomalies/ice_anom.rsi
+    layers:
+    - state: anom
+      map: ["enum.AnomalyVisualLayers.Base"]
+    - state: pulse
+      map: ["enum.AnomalyVisualLayers.Animated"]
+      visible: false
+  - type: PointLight
+    radius: 2.0
+    energy: 2.5
+    color: "#befaff"
+    castShadows: false
+  - type: Anomaly
+    anomalyContactDamage:
+      types:
+        Cold: 10
+  - type: IceAnomaly
+    supercriticalExplosion: Cryo
+    explosionTotalIntentsity: 600
+    explosionDropoff: 5
+    explosionMaxTileIntensity: 100
+    projectilePrototype: ProjectileIcicle
+    projectileDamage: # Values are scaled by severity
+      types:
+        Pierce: 50
+        Cold: 50
+        Structural: 30
```

Comment:
this should be on the projectile like i said earlier, but also, reduce the damage.

 This is enough to one shot crit a player at max severity. You could easily lower this to like 20 pierce, 20 cold, and 40 structural and still have it be a formidable projectile.